CONTRIBUTING.md

```diff
@@ -78,19 +78,25 @@ To contribute:
 
 - CI will run several build and functional testing checks as soon as the PR is
 approved by an Intel representative.
+  - By default, pre-commit activies (build/test) will not auto start when PR
+    is submitted for new contributors and non members of the project. If your
+    PR shows "workflow awaiting approval" then your assigned code reviewer or 
```

Comment:
Can you confirm that this message is shown to the PR author? I know that it's visible to the maintainers, but I'm not sure how it looks like for the outside contributors.